src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -67,11 +67,18 @@
                                     }
                                 </li>
                             }
+                            <!-- More icon and label in case of window minimization -->
+                            <li class="nav-item">
+                                <div [hidden]="!anyItemHidden" class="three-dots nav-link px-3" (click)="toggleDropdown()">
+                                    <fa-icon [fixedWidth]="true" [icon]="faEllipsis" class="ms-2 me-3" />
+                                    <span class="more" [hidden]="isNavbarCollapsed" [jhiTranslate]="'artemisApp.courseOverview.menu.more'"></span>
+                                </div>
+                            </li>
```

Comment:
I do not think that the comment is needed. A better way to show that this is the MoreIcon would be to add a component for the MoreIcon.

```suggestion
                            <li class="nav-item">
                                <div [hidden]="!anyItemHidden" class="three-dots nav-link px-3" (click)="toggleDropdown()">
                                    <fa-icon [fixedWidth]="true" [icon]="faEllipsis" class="ms-2 me-3" />
                                    <span class="more" [hidden]="isNavbarCollapsed" [jhiTranslate]="'artemisApp.courseOverview.menu.more'"></span>
                                </div>
                            </li>
```